### DISCUSSION TITLE
Can anyone explain me how to do the configuration?

### DISCUSSION THREAD

**Discussion by danielcolumbus** [fsaris/EspHome-AwoX-BLE-mesh-hub#46] [Q&A]
Can anyone explain me how to do the configuration? I've done everíthing but my MQTT integration can't find the light bulbs. Please help me!

**Comment by fsaris** [comment]
Is esphome able to connect to your mqtt server? (Should be visible in the esphome logs of your device).

Did you use the name from the example yaml?

Then try to listen to `awox-ble-mesh-hub/#` using a mqtt client and check if there are messages published.

Is esphome able to connect to one of the lights?  (Should be visible in the esphome logs of your device).

**Reply by danielcolumbus** [reply]
I have a few questions...
-In which log should I see the connection? In the log of the esp32 device?
-What do you mean by "the name from the example yaml"
-awox-ble-mesh-hub/# is a topic?
